share/CWW_E_ext_more.sed

```diff
@@ -0,0 +1,278 @@
+#  E SED of Coleman, Wu, and Weedman (1980)
```

Comment:
OK - I pushed a few commits that include the following:

1) Self-consistent thinning for the more/less thinned ones.
2) Keep them only in share/, not examples/data/.
3) Once they aren't in examples/data/, I didn't think getSEDs.py should be either.  So I moved it to devel/modules/.  Its location is noted in the comments of the SEDs themselves.